S/SFML/build_tarballs.jl

```diff
@@ -0,0 +1,89 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "SFML"
+version = v"2.5.1"
+
+# Collection of sources required to build SFML
+sources = [
+    GitSource(
+        "https://github.com/SFML/SFML.git",
+        "2f11710abc5aa478503a7ff3f9e654bd2078ebab",
+    )
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+# build SFML
+cd ${WORKSPACE}/srcdir
+
+cd SFML
+mkdir build && cd build
+
+CMAKE_FLAGS="-DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release"
+
+
+if [[ "${target}" == *-linux-* ]]; then
+    # I know I'm doing something evil here...
```

Comment:
It's not just evil, it's plainly wrong :wink:

Reply:
I know. Just wanna see whether it could fake BB2 or not. :innocent: